src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/BulkScanSasTokenCache.java

```diff
@@ -55,13 +54,15 @@ private String createSasToken(String containerName) {
         @Override
         public long expireAfterCreate(
             @NonNull String containerName,
-            @NonNull String sasToken, long currentTime) {
+            @NonNull String sasToken, long currentTime
+        ) {
             Map<String, String> map = parseQueryString(sasToken);
             return calculateTimeToExpire(
                 Constants.ISO_8601_UTC_DATE_FORMATTER.parse(
                     map.computeIfAbsent(
                         SAS_EXPIRY_TIME, key -> {
-                            throw new InvalidSasTokenException(EXPIRY_NOT_FOUND);
+                            throw new InvalidSasTokenException("Invalid SAS, "
+                                                                   + "the SAS expiration time parameter not found.");
```

Comment:
what was wrong with static string? or even static exception - the contents are not changing, it doesn't have any cause